study.md

```diff
@@ -95,11 +107,14 @@ would use to satisfy these requirements.
 In your own words, explain the role of application seed data.
 
 ```md
-<!-- your response here -->
+Seed data populates the database with an initial set of data you specify in
+db/seeds.rb.
 ```
 
 Should you use seeds to create data to experiment with during development?
 
 ```md
-<!-- your response here -->
+You could, but seeds are really just meant for adding necessary initial data.
+Creating data for testing can be done with gems that are better suited to that
+task (Faker, for example). 
 ```
```

Comment:
Nice! I'm on the fence with `Faker`, since it's an API for creating objects... which is what Ruby does already... ;)